pkg/flow/tracer_map.go

```diff
@@ -102,14 +102,13 @@ func (m *MapTracer) evictFlows(ctx context.Context, forceGC bool, forwardFlows c
 	flows := m.mapFetcher.LookupAndDeleteMap(m.metrics)
 	elapsed := time.Since(currentTime)
 	for flowKey, flowMetrics := range flows {
-		aggregatedMetrics := m.aggregate(flowMetrics)
 		// we ignore metrics that haven't been aggregated (e.g. all the mapped values are ignored)
```

Comment:
nit: irrelevant comment

Reply:
in fact, I don't quite understand what follows.. Do you?
```
		// we ignore metrics that haven't been aggregated (e.g. all the mapped values are ignored)
		if flowMetrics.EndMonoTimeTs == 0 {
			continue
		}
```
Are there actual cases where end time can be 0? Or was there, before this refactoring? I don't think so, but preferred to leave it as is ...

Reply:
fixed